lib/src/parser/recursive.dart

```diff
@@ -425,136 +443,23 @@ class RecursiveAstParser {
         properties: properties,
         references: references,
         bananas: bananas,
-        stars: stars,
+        stars: (uniqueStarAst == null ? [] : [uniqueStarAst]),
```

Comment:
we may want to have all the star asts inside the ElementAst, so that we can, for instance, autocomplete inside them before forcing the user to refactor into two elements with stars.